### ISSUE TITLE
Allow "multiple folders" to search for nested working copies

### ISSUE THREAD

**Issue by tlemo** [JohnstonCode/svn-scm#406]
- VSCode Version: 1.28.2
- OS Version: Debian Linux
- Extension Version: 1.45.0

## Issue ##

Regardless of the "multipleFolder" settings, the search seems to stop as soon as a .svn directory is found. The problem is that certain projects (ex. LLVM) have a source tree structure where work copies can be nested within other work copies.

```
  llvm/.svm
  llvm/tools/lldb/.svn
  ...
```
For these projects would be nice to let the search continue inside the working copies (otherwise only the top level working copy is detected)

Thanks.

**Comment by JohnstonCode** [comment]
Do you have `svn.multipleFolders.enabled` set to true?

**Comment by JohnstonCode** [comment]
Thanks for raising this

**Comment by tlemo** [comment]
> Do you have svn.multipleFolders.enabled set to true?

Yes, here are my settings (the ignore = [] doesn't really make any difference)

```
    "svn.multipleFolders.depth": 1024,
    "svn.multipleFolders.enabled": true,
    "svn.multipleFolders.ignore": []
```

**Comment by edgardmessias** [comment]
@tlemo , the problem is how extension detect the subfolder.
Current, the extension ignore all subfolder when find a `.svn` folder in parent folder

**Comment by tlemo** [comment]
@edgardmessias , thanks for the confirmation. Any chance the search can be extended to allow searching under working copies? (even if it's optional, controlled by a setting)

**Comment by edgardmessias** [comment]
@tlemo , yes, I talked to @JohnstonCode 

The question is: How check `llvm/tools/lldb` is not part of `llvm/`
In some cases, the repository uuid is same

Maybe, we can check the path, but is slow
While scanning, build the virtual URL and compare with URL of svn info
If diff, open the repository

Example: `http://mytest/llvm` for `llvm`
Scan subfolders with `.svn`
Found the `llvm/tools/lldb` with `.svn`
The virtual URL is `http://mytest/llvm/tools/lldb`
Is same url of svn info llvm/tools/lldb?
If `true`, ignore, if `false`, open repository

**Comment by edgardmessias** [comment]
@tlemo , is a good alternative, but in my work uses the svn 1.6 and it is a pain.

I will work on this in future

**Comment by tlemo** [comment]
Good question: I see that llvm/tools lists 'lldb' in its svn:ignore (and a few other subdirectories which are normally added as standalone working copies)

**Comment by tlemo** [comment]
BTW, in my particular case I'd be happy if there was an option to explicitly list the SVN subdirectories instead of the automatic search (since the directory structure is fairly stable and there are only a handful of nested working copies)

**Comment by edgardmessias** [comment]
Closed by #586
